test/unit/workers/worker_base_test.rb

```diff
@@ -0,0 +1,27 @@
+class WorkerBaseTest < ActiveSupport::TestCase
+  def self.worker_has_run!
+  end
+
+  class MyWorker < WorkerBase
+    def perform
+      WorkerBaseTest.worker_has_run!
+    end
+  end
+
+  test ".perform_async runs the job" do
+    self.class.expects(:worker_has_run!)
+    MyWorker.perform_async
+  end
+
+  test ".perform_async_in_queue runs the job in the specified queue" do
```

Comment:
if you decide to open `Sidekiq::Worker::ClassMethods`, this test will still be useful to have. it would be better if it doesn't have to worry about the internals of `Sidekiq::Worker`, by asserting using the [Sidekiq API](https://github.com/mperham/sidekiq/wiki/API):

```
assert_difference "Sidekiq::Queue.new('test-queue').size" do
  MyWorker.perform_async_in_queue('test_queue', example_arg)
end
```